pkgs/tools/networking/gp-saml-gui/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, python3Packages, fetchFromGitHub, gtk3, gobject-introspection, webkitgtk,
+  wrapGAppsHook, glib-networking }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gp-saml-gui";
+  version = "unstable-2020-10-13";
+
+  src = fetchFromGitHub {
+    owner = "dlenski";
+    repo = pname;
+    rev = version;
+    sha256 = "1yaqd8rf9lhahhdycpdrih70imp771b0ihs66clc7j7c3nxr8lym";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook webkitgtk gobject-introspection ];
+  buildInputs = [ gtk3 glib-networking ];
+
+  propagatedBuildInputs = with python3Packages; [ requests pygobject3 ];
+
+  meta = with lib; {
+    description = "A tool to authenticate to GlobalProtect VPN which uses SAML for authentication";
+    license = licenses.gpl3Only;
+    homepage = "https://github.com/dlenski/gp-saml-gui";
```

Comment:
maintainers is missing.

Reply:
I didn't know what to put here. I'm not sure if I can maintain this long-term. In such scenario, would it be worth it to cancel the PR?

Reply:
If you mean by long-term for the next month than we should think about closing this PR but if you can maintain it for a year this is fine.